OpenRA.Game.exe.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>

```

Comment:
I assume it will then not work when you compile from bleed and throw a thirdparty mod DLL in there.